README.md

```diff
@@ -358,34 +358,25 @@ Email [help@rainforestqa.com](mailto:help@rainforestqa.com) if you're having tro
 1. Push to the branch (`git push origin my-new-feature`)
 1. Create a new Pull Request
 
-## Release process
+## Development + release process:
 
-Check the [`.circleci/config.yml`](.circleci/config.yml) for the latest, but currently merging to master will build and deploy to the following Equinox channels:
-
-Tag | Channels
---------------------------------|-------------
-No tag | `dev`
-`vX.Y.Z-alpha.N` or `vX.Y.Z-beta.N` | `beta`, `dev`
-`vX.Y.Z` | `stable`, `beta`, `dev`
-
-### Development + release process is:
-
-#### Development PR
+### Development PR
 1. Branch from master
 1. Do work
 1. Open PR against master
 1. Merge to master
-#### Changelog PR
+### Changelog PR
 1. Branch from master to update `CHANGELOG.md` to include the commit hashes and release date
```

Comment:
Orthogonal to this change, but wondering if we should drop the requirement for a sha in the changelog, as it would let us release with one less PR?

Reply:
Maybe? It's nice to be able to easily group multiple smaller PRs into one update, and not that much of a hassle to open a separate release PR. But either way I'd rather not have that change in this PR.